spec/index.html

```diff
@@ -980,6 +980,18 @@ <h3>Examples</h3>
     <section id="canon-algo-algo">
       <h3>Algorithm</h3>
 
+      <p>The following algorithm will run with a minimal number of iterations in each step
+        for well formed <a>input datasets</a>.
```

Comment:
What does "well formed" means? Poison sets are also well formed, i.e., they are bona fide RDF graph datasets...

"average"? "general"? "practical?"

Reply:
Maybe. “Well behaved” or “typical”.

Reply:
I'd vote for "typical"

Reply:
```suggestion
        for typical <a>input datasets</a>.
```